web/src/views/InvestigationDetail.vue

```diff
@@ -488,8 +489,14 @@ export default defineComponent({
     });
 
     const selectedPins: Ref<Pin[]> = computed(() => store.state.selectedPins);
+    /**
+     * Create note cards for any pins without a child dataset.
+     */
```

Comment:
Use C++ style comments.

```suggestion
    // Create note cards for any pins without a child dataset.
```